src/main/java/core/basesyntax/service/operation/impl/TransactionHandlerImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.operation.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.operation.OperationHandler;
+import core.basesyntax.service.operation.OperationStrategy;
+import core.basesyntax.service.operation.TransactionHandler;
+import java.util.List;
+
+public class TransactionHandlerImpl implements TransactionHandler {
+
+    @Override
+    public void parse(List<FruitTransaction> fruitTransactionList,
+                      OperationStrategy operationStrategy) {
+        for (FruitTransaction fruitTransaction : fruitTransactionList) {
+            OperationHandler operationHandler = operationStrategy.get(fruitTransaction
+                    .getOperation());
+            operationHandler.operation(fruitTransaction);
+        }
+    }
+}
```

Comment:
the logic works correctly, but it is not a strategy - it is a service. Also, I would call it FruitService. And the proccesFruit method